docs/content/tutorials/quickstart.md

```diff
@@ -0,0 +1,273 @@
+---
+layout: doc_page
+---
+
+# Druid Quickstart
+
+In this quickstart, we will download Druid, set up it up on a single machine, load some data, and query the data.
+
+## Prerequisites
+
+You will need:
+
+  * Java 7 or higher
+  * Linux, Mac OS X, or other Unix-like OS (Windows is not supported)
+  * 8G of RAM  
```

Comment:
never tried it on single core machine, not sure if that would work. we should probably say at least 2 cores.